### PR TITLE
Add Ace ropes to the starting Arsenal.

### LINKED ISSUE #2172: [Change Request]: Add ACE Ropes to ACE Arsenal init
### Is the requested change related to a problem?

ACE Ropes can be used for Fast Roping and soon for Towing.
Also other mods are falling back on the ace ropes for many different things.

### Describe the solution you'd like.

Adding the ace ropes in its different lenghts to the Ace arsenal init should be an easy task.

### Alternative solutions

Spawn them randomly in civilian spawned vehicles inventory.

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by killerswin2**
## What type of PR is this.
1. [ ] Bug
2. [x] Change
3. [x] Enhancement

### What have you changed and why?
Information:
    Adds all ace ropes to the arsenal, players always unlock them. Ace always adds them to the vehicle after init.

### Please specify which Issue this PR Resolves.
closes #2172

### Please verify the following and ensure all checks are completed.

1. [x] Have you loaded the mission in LAN host?
2. [ ] Have you loaded the mission on a dedicated server?

### Is further testing or are further changes required?
1. [x] No
2. [ ] Yes (Please provide further detail below.)

### How can the changes be tested?
Steps: 

********************************************************
Notes:

### CODE REVIEW

**Review by Lazejun — COMMENTED** [review]
You might want to prevent undercover towing with ace before this is merged.

File: A3A/addons/core/functions/init/fn_initClient.sqf [hunk 1]
```diff
@@ -283,6 +283,11 @@ if (A3A_hasACE) then {
         params ["_explosive","_dir","_pitch","_unit"];
         if (_unit == player) then { player setCaptive false };
     }] call CBA_fnc_addEventHandler;
+
+    ["ace_towing_ropeDeployed", {
```
Comment (jaj22) [inline A3A/addons/core/functions/init/fn_initClient.sqf:287]: I can't find this event anywhere in the ACE codebase. What am I missing?

Comment (killerswin2) [reply A3A/addons/core/functions/init/fn_initClient.sqf:287]: It doesn't exist. I had to make a pr to resolve this

Comment (killerswin2) [reply A3A/addons/core/functions/init/fn_initClient.sqf:287]: https://github.com/acemod/ACE3/pull/9178

Comment (jaj22) [reply A3A/addons/core/functions/init/fn_initClient.sqf:287]: Hah ok, good job.

File: A3A/addons/core/functions/init/fn_initACE.sqf [hunk 2]
```diff
@@ -39,6 +39,11 @@ if (A3A_hasACEMedical) then {
     if (captive player && _unit == player) then { player setCaptive false }; 
 }] call CBA_fnc_addEventHandler;
 
+["ace_towing_ropeDeployed", {
+        params ["_unit", "_target", "_ropeClass"]
+        if (_unit == player) then { player setCaptive false };
```
Comment (rautamiekka) [inline A3A/addons/core/functions/init/fn_initACE.sqf]:
```suggestion
    params ["_unit", "_target", "_ropeClass"]
    if (_unit == player) then { player setCaptive false };
```

**Review by rautamiekka — CHANGES_REQUESTED** [review]

**Review by jaj22 — APPROVED** [review]

**Comment by jaj22** [comment]
Oh yeah, you do have a double-tab there :P

**Pull request merged**